sos/plugins/samba.py

```diff
@@ -27,7 +27,11 @@ def setup(self):
             "/etc/samba/smb.conf",
             "/etc/samba/lmhosts",
             "/var/log/samba/log.smbd",
-            "/var/log/samba/log.nmbd"
+            "/var/log/samba/log.nmbd",
+            "/var/log/samba/log.winbindd",
+            "/var/log/samba/log.winbindd-idmap",
+            "/var/log/samba/log.winbindd-dc-connet",
+            "/var/log/samba/log.wb-*"
```

Comment:
Ack, although while we are making these changes we should probably also teach `samba.py` to obey the global `--log-size` - that way, users can request all logs, but with a per-log-file size limit (that effectively collects `tail -c $BYTES` for each matching file).